scripts/irods/test/resource_suite.py

```diff
@@ -1260,7 +1261,7 @@ def assert_permissions_on_data_object_for_user(username, logical_path, permissio
             owner_user.assert_icommand(['ichmod', '-r', 'read', readonly_user.username, os.path.dirname(logical_path)])
 
             # Ensure that the read-only user has read-only permission on the data object.
-            assert_permissions_on_data_object_for_user(readonly_user.username, logical_path, 1050)
+            self.assert_permissions_on_data_object_for_user(readonly_user.username, logical_path, 1050)
```

Comment:
not required of course - consider 1050 -> READ_OBJECT or something like that.

Reply:
Done